.github/workflows/linux-flatpak.yml

```diff
@@ -0,0 +1,85 @@
+name: Linux (Flatpak)
+
+on:
+  workflow_dispatch:
+    inputs:
+      myCommit:
+        description: Commit SHA1
+        required: false
+        default: ''
+        type: string
+      releaseTag:
+        description: Release tag
+        required: false
+        default: ''
+        type: string
+  workflow_call:
+    inputs:
+      myCommit:
+        description: Commit SHA1
+        required: false
+        default: ''
+        type: string
+      releaseTag:
+        description: Release tag
+        required: false
+        default: ''
+        type: string
+  push: null
+  pull_request: null
+  release:
+    types:
+      - published
+
+jobs:
+  build-flatpak-bundle:
+    name: "Build flatpak bundle"
+    runs-on: ubuntu-latest
+    container:
+      image: bilelmoussaoui/flatpak-github-actions:freedesktop-23.08
+      options: --privileged
+    strategy:
+      matrix:
+        arch:
+          - x86_64
+          - arm64
+
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        submodules: false
+
+    - name: Set up Docker
+      if: ${{ matrix.arch != 'x86_64' }}
+      run: |
+        dnf install -y docker
+
+    - name: Set up QEMU
+      if: ${{ matrix.arch != 'x86_64' }}
+      id: qemu
+      uses: docker/setup-qemu-action@v2
+      with:
+        platforms: arm64
+
+    - name: Run Flatpak build
+      uses: flatpak/flatpak-github-actions/flatpak-builder@v6
+      with:
+        bundle: io.github.openenroth.openenroth_${{inputs.releaseTag}}_${{ matrix.arch }}.flatpak
+        manifest-path: distribution/linux/flatpak/io.github.openenroth.openenroth.yml
+        cache-key: flatpak-builder-${{ github.sha }}_${{ matrix.arch }}
```

Comment:
Docs mention that default value is `flatpak-builder-${sha256(manifestPath)}`. Should't we use this instead of `github.sha`?